modules/draco/src/lib/draco-parser.ts

```diff
@@ -289,6 +293,7 @@ export default class DracoParser {
 
     for (const loaderAttribute of Object.values(loaderData.attributes)) {
       const attributeName = this._deduceAttributeName(loaderAttribute, options);
+      loaderAttribute.name = attributeName;
```

Comment:
`loaderData` is intended to be very close to the Draco formats original data. Since Draco attributes don't really have names (we "invent" names for the attributes), I didn't add them here. Instead I was adding a separate name map.